src/main/java/core/basesyntax/db/dto/StorageOperationDto.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax.db.dto;
+
+public class StorageOperationDto {
+    private final String type;
+    private final StorageItemDto storageItem;
```

Comment:
I'd remove Dto from naming.
DTO is used when we accept objects from outside or passing objects to external user in order to not to be tight to internal implementation and possibly hide some aspects of it.
Here we reading file by ourselves and there is no reason to wrap anything and by fact it's just a model